src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -379,16 +907,37 @@
 		"sourcePath" : "log.sagebase.org/certifiedUserPassingRecords/",
 		"tableDescriptor": {
 			"name": "certifiedquizquestionrecords",
-			"columns": {
-				"response_id": "bigint",
-				"question_index": "bigint",
-				"is_correct": "boolean",
-				"stack": "string",
-				"instance": "string"
+			"description" : "This table contains records of the questions of certified quiz taken bu Synapse user.",
```

Comment:
Please improve the description: Based on the fields, the table contains the result of a user taking the quiz.
Also, fix the grammar and typo.
Can we use the phrase "certification quiz" rather than "certified quiz". It's the *user* who is certified, not the quiz.